app.js

```diff
@@ -0,0 +1,157 @@
+//jshint esversion: 6
+const express = require("express");
+const bodyParser = require("body-parser");
+const mongoose = require("mongoose");
+const redisClient = require("redis").createClient;
+const redis = redisClient(6379, 'redis-server');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
+app.use(express.static("public"));
+//let studentMap = new Map();
+
+mongoose.connect("mongodb://mongo:27017/mongorediscomposeDB", {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  useFindAndModify: false
+}, function(err) {
+  if (!err) {
+    console.log("successfully connected");
+  }
+});
+
+redis.on("connect", function() {
+  console.log('connected to Redis');
```

Comment:
Inconsistent logging, at some places the log messages start with single quotes, at other places they start with double quotes.

At some place they start with a capital letter at other places they start with a small case.

You can try to see and implement linting tools in your application so you have a consistent pattern everywhere, else it confuses the new person. It will at-least help in solving the first problem. Second one you will have to keep in mind.